tasks/3_is-boolean/is-boolean.js

```diff
@@ -0,0 +1,8 @@
+function isBoolean(value) {
+  if (typeof(value) === 'boolean') {
```

Comment:
`typeof(value) === 'boolean'` itself returns `boolean`.
This is a bad practice:
``` javascript
if(true){
 return true;
} else {
 return false;
}```